src/main/java/com/example/universe/simulator/entityservice/services/PlanetService.java

```diff
@@ -0,0 +1,44 @@
+package com.example.universe.simulator.entityservice.services;
+
+import com.example.universe.simulator.entityservice.entities.Planet;
+import com.example.universe.simulator.entityservice.exception.AppException;
+import com.example.universe.simulator.entityservice.exception.ErrorCodeType;
+import com.example.universe.simulator.entityservice.repositories.PlanetRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class PlanetService {
+
+    private final PlanetRepository repo;
+
+    public Planet get(UUID id) throws AppException {
+        return repo.findById(id).orElseThrow(() -> new AppException(ErrorCodeType.ENTITY_NOT_FOUND));
+    }
+
+    public List<Planet> getList() {
+        return repo.findAll();
+    }
+
+    @Transactional
+    public Planet add(Planet planet) {
+        return repo.save(planet);
+    }
+
+    @Transactional
+    public Planet update(Planet planet) throws AppException {
+        get(planet.getId());
```

Comment:
Replace this with validate method, where you check 3 things. 1. if entity id exists (only for update operation) 2. if entity name is unique 3. if star id exists. Check how it's done in case of StarService.